libcontainer/dmz/dmz_linux.go

```diff
@@ -1,4 +1,5 @@
-//go:build !runc_nodmz
+//go:build (386 || amd64 || arm || arm64 || loong64 || ppc64le || riscv64 || s390x) && !runc_nodmz
+// +build 386 amd64 arm arm64 loong64 ppc64le riscv64 s390x
```

Comment:
As a side note, I'm not sure why we still have old-style +build tags. Those are not needed since Go 1.18 or so.

Will open a PR to remove those.